enterprise-beans/4.0/_index.md

```diff
@@ -1,23 +1,28 @@
 ---
-title: "Jakarta Enterprise Beans 4.0 (milestone)"
-date: 2020-06-05
+title: "Jakarta Enterprise Beans 4.0"
+date: 2020-08-26
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Enterprise Beans defines an architecture for the development and deployment of component-based business applications.
 
 * [Jakarta Enterprise Beans 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan)
```

Comment:
You need to complete and schedule a release review per the checklist in the opening Comment on this PR.  I also pinged you about this on Slack. Thanks!

Reply:
EMO/PMC tasks should be done now.  I don't know if I did them correctly, but I expect the EMO will provide guidance if not.

Reply:
Thanks.